docs/source/settings.rst

```diff
@@ -55,8 +55,8 @@ bind
 
 The socket to bind.
 
-A string of the form: ``HOST``, ``HOST:PORT``, ``unix:PATH``. An IP is
-a valid ``HOST``.
+A string of the form: ``HOST``, ``HOST:PORT``, ``unix:PATH``, ``fd://FD``. An 
+IP is a valid ``HOST``.
```

Comment:
Could you make this change in `gunicorn/config.py` instead, regenerate `docs/source/settings.rst` by running `make -C docs html`?